src/main/java/thrift/model/util/CurrencyUtil.java

```diff
@@ -32,7 +32,7 @@
      * @return Amount in target currency as a double.
      */
     public static double convertFromDefaultCurrency(Map<String, Double> currencyMappings, double value,
-        String currency) {
+            String currency) {
```

Comment:
Does checkStyle complain that newline indent isn't 8 spaces? If not, ignore this comment.

Reply:
There are 8 spaces.